test/cmdlineTests/strict_asm_eof_container_prague/args

```diff
@@ -1 +1 @@
- --strict-assembly --experimental-eof-version 1 --evm-version prague --bin
+ --strict-assembly --experimental-eof-version 1 --evm-version prague
```

Comment:
You can use `--debug-info none` to suppress the debug comments, since they're just noise here.

Also, better to select outputs explicitly, since the fact that they're printed by default is just a historical artifact and will change in next breaking release (#12230).

```suggestion
--strict-assembly --experimental-eof-version 1 --evm-version prague --asm --ir-optimized --bin --debug-info none
```